### PR TITLE
serverListFilters configuration does not take effect

### PR DESCRIPTION
**Pull request by fanjiwang1992**
serverListFilters configuration does not take effect

### CODE REVIEW

**Comment by tianxiaoliang** [comment]
thanks, please add unit test

**Comment by fanjiwang1992** [comment]
> thanks, please add unit test

done

File: control/servicecomb/transfer_test.go [hunk 1]
```diff
@@ -28,6 +29,9 @@ func TestSaveToLBCache(t *testing.T) {
 	})
 	c, _ := servicecomb.LBConfigCache.Get("test")
 	assert.Equal(t, loadbalancer.StrategyRoundRobin, c.(control.LoadBalancingConfig).Strategy)
+	c, _ = servicecomb.LBConfigCache.Get("")
```
Comment (tianxiaoliang) [inline control/servicecomb/transfer_test.go:32]: 用t.Run隔离不同test case， 已维护的case name来增加可维护性

Comment (fanjiwang1992) [reply control/servicecomb/transfer_test.go]: done

**Review by tianxiaoliang — APPROVED** [review]

**Pull request merged**